CODE_OF_CONDUCT.md

```diff
@@ -2,3 +2,14 @@
 
 Ambassador follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
+As contributors and maintainers of any datawire.io or Ambassador.io project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
```

Comment:
Maybe "the Ambassador project" is sufficient here -- this will be the CoC just for Ambassador, after all.

Reply:
Ok so just remove project and leave Ambassador? Should I remove Datawire as well?